_layouts/docs.html

```diff
@@ -41,7 +40,15 @@
           <div class="column column-content">
             <button class="button is-rounded" id="table-of-contents" onclick="tocToggler()">Table of contents</button>
             <div class="content">
-                <h1>{{page.title}}</h1>
+              <!-- If it is the landing page for a product, then print out the currently selected version for the product. -->
+              <!-- A page is considered to be the landing page if the path to it is <product id>/<product version>/index.md -->
+              <!-- Any longer path is considered a sub page, which should not have the product version printed out -->
+              {% assign pathElements = page.path | split: "/"%}
+              {% if pathElements.size == 3 %}
+              <h1>{{page.title}} {{page.version}}</h1>
+              {% else %}
+              <h1>{{page.title}}</h1>
+              {% endif %}
```

Comment:
Maybe you can find a more elegant way of doing this? Tested locally and it works.

```suggestion
              <!-- For each landing page, print out the selected version for the product (unless it's a Community version). -->
              <!-- A landing page has a path with format: <product-name>/<product-version>/index.md -->
              <!-- A longer path is considered a sub page, which should not have the version printed out. -->
              {% assign pathElements = page.path | split: "/"%}
              {% if pathElements.size == 3 %}
              {% unless page.version == 'community' %}
              <h1>{{page.title}} {{page.version}}</h1>
              {% else %}
              <h1>{{page.title}}</h1>
              {% endunless %}
              {% else %}
              <h1>{{page.title}}</h1>
              {% endif %}
```